src/parser.js

```diff
@@ -505,7 +506,7 @@ define([
             // attempt to support sloppy hand-written forms
             nodeId = noPop ? el.attr('bind') : el.popAttr('bind');
             if (nodeId) {
-                pathToTry = processPath(nodeId);
+                pathToTry = processPath(nodeId, rootNodeName, form);
```

Comment:
I don't think this would have ever worked properly, but it comes from a commit directly to master with no PR or tests.